lib/logstash/codecs/json.rb

```diff
@@ -29,13 +43,20 @@ class LogStash::Codecs::JSON < LogStash::Codecs::Base
   # For nxlog users, you may to set this to "CP1252".
   config :charset, :validate => ::Encoding.name_list, :default => "UTF-8"
 
+  # Defines a target field for placing decoded fields.
+  # If this setting is omitted, data gets stored at the root (top level) of the event.
+  # The target is only relevant while decoding data into a new event.
+  #
+  # NOTE: if the `target` field already exists, it will be overwritten!
```

Comment:
Since we are the ones _creating_ the event, we don't need to have a caveat about the field already being present.

```suggestion
```